src/components/home/Home.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import i18next from "i18next";
+import { useTranslation } from "react-i18next";
+
+export default function Home() {
+  const { t } = useTranslation();
+
+  function setLanguage(language) {
+    i18next.changeLanguage(language);
+  }
+
+  return (
```

Comment:
even for basic setup you should move this out into a different component, such as languagedropdown component. eventually you have to move it out anyway, so why not do it now :)